scene/main/scene_tree.cpp

```diff
@@ -1605,7 +1620,8 @@ void SceneTree::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_pause", "enable"), &SceneTree::set_pause);
 	ClassDB::bind_method(D_METHOD("is_paused"), &SceneTree::is_paused);
 
-	ClassDB::bind_method(D_METHOD("create_timer", "time_sec", "process_always", "process_in_physics", "ignore_time_scale"), &SceneTree::create_timer, DEFVAL(true), DEFVAL(false), DEFVAL(false));
+	ClassDB::bind_method(D_METHOD("create_timer", "time_sec", "process_always", "process_in_physics", "ignore_time_scale", "owner"), &SceneTree::create_timer, DEFVAL(true), DEFVAL(false), DEFVAL(false));
```

Comment:
```suggestion
	ClassDB::bind_method(D_METHOD("create_timer", "time_sec", "process_always", "process_in_physics", "ignore_time_scale", "owner"), &SceneTree::create_timer, DEFVAL(true), DEFVAL(false), DEFVAL(false), DEFVAL(nullptr));
```
And you need to register a compatibility method, see [here](https://docs.godotengine.org/en/latest/contributing/development/handling_compatibility_breakages.html)

Otherwise there's no concerns left, will check the functionality later!